File: C#1/03. Operators-and-Expressions/Rectangles/Rectangles.cs
using System;
using System.Globalization;

    class Rectangles
    {
        static void Main(string[] args)
        {
        double width = double.Parse(Console.ReadLine());
        double height = double.Parse(Console.ReadLine());
        double perimeter = 2 * (width + height);
        double area = width * height;
        string a = String.Format(CultureInfo.InvariantCulture, "{0:0.00}", area);
        string b = String.Format(CultureInfo.InvariantCulture, "{0:0.00}", perimeter);
        
        Console.WriteLine("{0} {1}", a, b);
    }
    }

File: JavaScript/02. Operators and Expressions/Rectangles.js
function solve(args) {
    var a = +args[0];
    var b = +args[1];

    var area = a * b;
    var perimeter = 2 * a + 2 * b;

    console.log(area.toFixed(2) + ' ' + perimeter.toFixed(2));
}

solve(['2.5', '3']);